flocker/control/_persistence.py

```diff
@@ -4,11 +4,12 @@
 Persistence of cluster configuration.
 """
 
-from json import dumps, loads
-from uuid import UUID
+from base64 import b16encode
 from calendar import timegm
 from datetime import datetime
-from hashlib import sha256
+from json import dumps, loads
+from mmh3 import hash_bytes
```

Comment:
yea that's definitely better, 👍 

Also apologies about sorting imports. Old habit.